pkg/sensors/tracing/policyhandler_linux.go

```diff
@@ -126,6 +126,27 @@ func (h policyHandler) PolicyHandler(
 			"policy", tracingpolicy.TpLongname(policy),
 			"sensor", name,
 		)
+
+		// Check if any selector uses NotifyEnforcer action
```

Comment:
The check makes sense to me. Can you add it to the `preValidateKprobes` function?
I think it's better to have all validation in a single place.

Reply:
Yeah makes sense! Thanks